FirebaseMessaging/Sources/FIRMessagingUtilities.m

```diff
@@ -114,28 +114,6 @@ BOOL FIRMessagingIsWatchKitExtension(void) {
 #endif
 }
 
-uint64_t FIRMessagingGetFreeDiskSpaceInMB(void) {
-  NSError *error;
-  NSArray *paths =
-      NSSearchPathForDirectoriesInDomains(FIRMessagingSupportedDirectory(), NSUserDomainMask, YES);
-
-  NSDictionary *attributesMap =
-      [[NSFileManager defaultManager] attributesOfFileSystemForPath:[paths lastObject]
-                                                              error:&error];
-  if (attributesMap) {
-    uint64_t totalSizeInBytes __unused = [attributesMap[NSFileSystemSize] longLongValue];
-    uint64_t freeSizeInBytes = [attributesMap[NSFileSystemFreeSize] longLongValue];
-    FIRMessagingLoggerDebug(
-        kFIRMessagingMessageCodeUtilities001, @"Device has capacity %llu MB with %llu MB free.",
-        totalSizeInBytes / kBytesToMegabytesDivisor, freeSizeInBytes / kBytesToMegabytesDivisor);
```

Comment:
Looks like `kBytesToMegabyteDivisor` should be removed to fix the warning causing the failed CI jobs.

Reply:
Done.